visidata/clipboard.py

```diff
@@ -113,6 +113,8 @@ def pasteFromClipboard(vd, cols, rows):
 
 @Sheet.api
 def delete_row(sheet, rowidx):
+    if len(sheet.rows) == 0:
```

Comment:
This looks like a friendly move 👍 . Makes me wonder though if we need to account for cases where `sheet.rows` is `None` or `UNLOADED` 🤔. If so, maybe... 

```suggestion
    if not sheet.rows:
```

In any case yay for friendlier errors 🎉

Reply:
Ah good idea, @ajkerrigan!  It's unlikely but as long as we're here..

Reply:
Good catch. So, should it handle UNLOADED, should it say `if not sheet.rows or sheet.rows is UNLOADED`?

Reply:
`UNLOADED` is False-y, so `if not sheet.rows` should suffice for all cases.